2009-flu-uk.qmd

```diff
@@ -1 +1,437 @@
+  if (before) {
+    par(mar = c(4, 4, .1, .1))
+  }
+})
+```
+## Introduction
+
+The 2009 A/H1N1 influenza pandemic [@ghani_early_2009] posed a significant public health challenge in England and Wales, requiring rapid and evidence-based decision-making to mitigate its impact [@Baguelin2010a]. This chapter explores the use of `odin` and `monty` to model the dynamics of the 2009 A/H1N1 influenza pandemic in England and Wales, incorporating changes in social contacts during holiday periods. We demonstrate how to construct, fit, and analyse a compartmental model to infer key epidemiological parameters.
+
+### Objective of this chapter
+
+The chapter is structured to guide the reader through an entire modelling pipeline, from data preparation and visualisation to model construction, parameter inference, and validation. Along the way, we highlight the integration of real-world data, discuss challenges in linking models to observed cases, and showcase the use of Bayesian methods for robust parameter estimation.
+
+### Overview of the pipeline
+
+This chapter illustrates how to:
+
+1. Construct an SEIR transmission model with time-varying contact rate using `odin`
+2. Implement a (probabilistic) observation model in `odin`
+3. Derive a likelihood based on observations using `dust` and convert it to `monty`
+4. Built a prior distribution for the model parameters using the `monty` DSL
+3. Run Bayesian inference using adaptive Markov chain Monte Carlo (MCMC) with `monty`
+4. Analyse and interpret the MCMC results.
+
+### Key parameters of interest
+
+We are interested in particular in two epidemiological parameters:
+
+   - The basic reproduction number $R_0$ of the novel influenza strain;  
+   - The ascertainment probability, i.e., the proportion of total infections reported as cases.
+
+## Data preparation and visualisation
+
+Data is the number of weekly cases of A/H1N1 pandemic cases from June 2009, and initially broken up into seven age-groups. For simplicity, we aggregate the data into a single time series for this analysis.
+
+### Loading data
+
+The data file is taken from the supplement material in [@endo_introduction_2019] and can be found [here](https://github.com/akira-endo/Intro-PMCMC/blob/master/assets/andre_estimates_21_02.txt).
+
+We will fit our model to all cases aggregated so we need to load our data, add the age-group, and then build a dataframe (a table where each rows is an observations of certains variables in the columns) with 'Days' and 'cases'. We use the `dplyr` package, a popular R package for data handling, with the '%>%' ('pipe') operator but other methods are possible.  
+
+```{r}
+library(dplyr)
+
+# Read the data
+original_data <- read.csv(file = "data/andre_estimates_21_02.txt", sep = "\t") %>% rowSums()
+
+# Group all age groups
+data <- data.frame(Cases = original_data) %>%
+  mutate(Days = seq(7, by = 7, length.out = n())) %>%
+  dplyr::select(Days, Cases)
```

Comment:
I'd suggest maybe using lowercase `days` and `cases` throughout

Reply:
Done.